app/screens/galoy-address-screen/set-address-modal.tsx

```diff
@@ -159,6 +161,12 @@ export const SetAddressModal = ({ modalVisible, toggleModal }: SetAddressModalPr
     setError("")
     setAddress(value)
   }
+
+  const usernameSuffix =
+    appConfig.galoyInstance.name.toLowerCase() === "staging"
```

Comment:
this value should be passed from the graphql layer. I think it's already available

Reply:
fixed here https://github.com/GaloyMoney/galoy-mobile/pull/805/commits/128c613d54fe5a0cc604ec914f430f7fc44137b5